tfhe/src/c_api/shortint/destroy.rs

```diff
@@ -8,7 +8,10 @@ use super::{
 };
 
 #[no_mangle]
-pub unsafe extern "C" fn destroy_shortint_client_key(client_key: *mut ShortintClientKey) -> c_int {
+pub unsafe extern "C" fn shortint_destroy_client_key(client_key: *mut ShortintClientKey) -> c_int {
+    if client_key.is_null() {
+        return 0;
+    }
```

Comment:
if at some point it makes sense to start macro-izing these modules feel free to do so !